homeassistant/components/cisco_webex_teams/notify.py

```diff
@@ -0,0 +1,57 @@
+"""Cisco Webex Teams notify component."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.notify import (
+    PLATFORM_SCHEMA, BaseNotificationService, ATTR_TITLE)
+from homeassistant.const import (CONF_TOKEN)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['webexteamssdk==1.1.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ROOM_ID = 'room_id'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_TOKEN): cv.string,
+    vol.Required(CONF_ROOM_ID): cv.string,
+})
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the CiscoWebexTeams notification service."""
+    from webexteamssdk import WebexTeamsAPI
+    client = WebexTeamsAPI(access_token=config[CONF_TOKEN])
+    # Validate the token & room_id
+    # pylint:disable=expression-not-assigned
+    client.rooms.get(config[CONF_ROOM_ID])
```

Comment:
Wrap this in try... except and catch `ApiError` imported from the library. Log error and `return None` in the except block.

Reply:
Good idea. Done.